backend/core/forms.py

```diff
@@ -87,5 +87,11 @@ def get_permitted_teams_for_user(user) -> list:
             any_perm=True,
         ):
             teams.append(team.name)
-        
+
         return sorted(set(teams))
+
+
+class DeleteAndRegenerateCardsForm(forms.Form):
+    def clean(self):
+        cleaned_data = super().clean()
+        return cleaned_data
```

Comment:
This doesn't seem necessary since you are just doing what's already being done automatically without anything custom. If you want to have the form for consistency's sake or so that you can use a class-based view, maybe you can just have the form as:
```
class DeleteAndRegenerateCardsForm(forms.Form):
	pass
```